css/src/tokens/ratios.scss

```diff
@@ -0,0 +1,10 @@
+/**
+ * @sass-export-section="ratios"
+ */
+
+$ratios: (
+	'16-9': '16 / 9',
+	'4-3': '4 / 3',
+	'square': '1 / 1',
+	'vertical': '9 / 16'
```

Comment:
This should probably be `9-16` instead of `vertical`, seeing as a hypothetical `3 / 4` would also count as vertical.

I think `square` is the aspect ratio where a non-numeric name makes sense to me.